Sources/ImageCarouselViewController.swift

```diff
@@ -148,16 +164,7 @@ class ImageCarouselViewController:UIPageViewController, ImageViewerTransitionVie
 
     @objc
     private func dismiss(_ sender:UIBarButtonItem) {
-        dismissMe(completion: nil)
-    }
-    
-    public func dismissMe(completion: (() -> Void)? = nil) {
-        sourceView?.alpha = 1.0
-        UIView.animate(withDuration: 0.235, animations: {
-            self.view.alpha = 0.0
-        }) { _ in
-            self.dismiss(animated: false, completion: completion)
-        }
+        self.dismiss(animated: true, completion: nil)
```

Comment:
This actually allows for the animation transition to run and it looks really nice!